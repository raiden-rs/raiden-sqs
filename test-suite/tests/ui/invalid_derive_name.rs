use raiden_sqs::*;
use serde::Serialize;

#[derive(RaidenCloudPubSub, Serialize)]
pub struct GreetingMessage {
    name: String,
}

fn main() {}
