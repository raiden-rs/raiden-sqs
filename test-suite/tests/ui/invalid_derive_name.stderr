error: cannot find derive macro `RaidenCloudPubSub` in this scope
 --> $DIR/invalid_derive_name.rs:4:10
  |
4 | #[derive(RaidenCloudPubSub, Serialize)]
  |          ^^^^^^^^^^^^^^^^^

warning: unused import: `raiden_sqs::*`
 --> $DIR/invalid_derive_name.rs:1:5
  |
1 | use raiden_sqs::*;
  |     ^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
